src/NATS.Client.KeyValueStore/INatsKVStore.cs

```diff
@@ -72,7 +72,7 @@ public interface INatsKVStore
     ValueTask<NatsKVEntry<T>> GetEntryAsync<T>(string key, ulong revision = default, INatsDeserialize<T>? serializer = default, CancellationToken cancellationToken = default);
 
     /// <summary>
-    /// Start a watcher for specific keys
+    /// Start a watcher for a specific key
```

Comment:
I think plural is still correct here since a wildcard may be used.

Reply:
I adjusted this back. Although, it reads goofy to me. The param is called "key" singular, but the comment says "keys" to represent a wildcard. I have no preference, but my OCD is triggered.

Reply:
maybe we keep your edit and add something in parenthesis or a remark that wildcards can be used?